### DISCUSSION TITLE
Issues with the AI Deck

### DISCUSSION THREAD

**Discussion by marlos1494** [bitcraze/discussions#1501] [Q&A]
Hello,

I am new to Crazyflie and recently got an AI Deck. I've been trying to get it to work without success. Initially, I tried to connect it as an access point, and the network did appear. Then, I tried to follow the tutorial step by step from https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/ . After updating the Crazyflie firmware with the ai deck attached, the network no longer appears as an access point. I'm not sure if this is normal.

I want to clarify that I did all the flashing over WiFi, not using JTAG.

When I check the console in cfclient, this is what I see:

SYS: ----------------------------
SYS: Crazyflie 2.1 is up and running!
SYS: Build 0:e527895e8c6e (2023.11) CLEAN
SYS: I am 0x2032353241335019001C0034 and I have 1024KB of flash!
CFGBLK: v1, verification [OK]
DECK_CORE: 1 deck(s) found
DECK_CORE: Calling INIT on driver bcAI for deck 0
CPX-EXT-ROUTER: Waiting for CPX External router initialization
CPX-EXT-ROUTER: CPX External router initialized, CPX_VERSION: 0
AIDECK: Not setting up WiFi
IMU: BMI088: Using I2C interface.
IMU: BMI088 Gyro connection [OK].
IMU: BMI088 Accel connection [OK]
IMU: BMP388 I2C connection [OK]
ESTIMATOR: Using Complementary (1) estimator
CONTROLLER: Using PID (1) controller
MTR-DRV: Using brushed motor driver
SYS: About to run tests in system.c.
SYS: NRF51 version: 2023.11
EEPROM: I2C connection [OK].
STORAGE: Storage check [OK].
IMU: BMI088 gyro self-test [OK]
DECK_CORE: Deck 0 test [OK].
SYS: Self test passed!
STAB: Wait for sensor calibration...
SYS: Free heap: 14592 bytes
CPX: ESP32: I (435) SYS: Initialized
CPX: ESP32: W (435) ROUTER: Cannot route from GAP8 [0x00] to [0x00]
CPX: ESP32: I (435) DISCOVERY: Hostname is aideck-04AB18
CPX: ESP32: I (435) DISCOVERY: MAC (STA) is 78:e3:6d:4:ab:18
STAB: Starting stabilizer loop
SUP: Can not fly
SUP: Ready to fly

I'm not sure why the WiFi is not setting up and there seems to be a routing issue with the CPX. Could you please help me troubleshoot this issue? Any guidance would be greatly appreciated.

Thank you!

**Comment by knmcguire** [comment]
Hi! This is indeed strange and seems like a mismatch in communication. Have you updated the Nina module as required in the getting started guide? https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/#update-crazyflie-and-aideck-firmware

**Comment by panosdiavo** [comment]
Hello guys,
I am trying to run the opencv-viewer.py from the examples in order to do the pose estimation of an ArUco marker and I keep getting the same message socket connected, and then nothing happens. Can anybody help me with this?

panosdiavo@panosdiavo-HP-Laptop-15s-eq1xxx:~/Downloads/aideck-gap8-examples/examples/other/wifi-img-streamer$ python3 opencv-viewer.py
Connecting to socket on 192.168.4.1:5000...
Socket connected

**Reply by gemenerik** [reply]
Please check out [this issue](https://github.com/bitcraze/aideck-gap8-examples/issues/150) and give the workaround a shot. Let me know how it goes.
